### ISSUE TITLE
Add a upload image feature in website

### ISSUE THREAD

**Issue by 7saikat7** [FOSS-Cell-GECPKD/pixelvibe#411]
you have to build a feature to let the users upload images and save it , later on the project we will be working with this uploaded image so keep it in mind.
Image models will have fields like , image,name,date-uploaded etc.

**Comment by shreyjha2309** [comment]
I'll try to do this one

**Comment by shreyjha2309** [comment]
Fixes #411

**Comment by 7saikat7** [comment]
@shreyjha2309  are you working on it ?? on new pr?

**Comment by shreyjha2309** [comment]
Yes, yes. College restarted and some other stuff so was busy a bit, will be working soon

**Comment by 7saikat7** [comment]
actually, this feature is there named Gallary,you can check it out so u can modify the model, link it with the Primary key with the logged-in user , rather than implementing a totally new one

**Comment by shreyjha2309** [comment]
> actually, this feature is there named Gallary,you can check it out so u can modify the model, link it with the Primary key with the logged-in user , rather than implementing a totally new one

ahh alright I also thought the same, I'll remove the new model then
